contrib/gcppubsub/pkg/dispatcher/dispatcher/reconcile.go

```diff
@@ -63,6 +66,9 @@ type reconciler struct {
 	// function must be called when we no longer want that subscription to be active. Logically it
 	// is a map from Channel name to Subscription name to CancelFunc.
 	subscriptions map[channelName]map[subscriptionName]context.CancelFunc
+
+	// to limit the pace at which we nack a message when it could not be dispatched
```

Comment:
Comments should be full sentences, starting with capital letter and ending with punctuation.

Reply:
Done! copied similar format of the above comments, with the variable name first and what it does.